frontend/api/fetchLifterNames/fetchLifterNames.ts

```diff
@@ -0,0 +1,18 @@
+import { LifterSearchList } from "./fetchLifterNamesTypes"
+
+export default async function fetchLifterNames(nameQuery: string): Promise<LifterSearchList> {
+  if (nameQuery.length < 3) {
+    return {names: []}
+  }
+
+  try {
+    const response = await fetch(`https://api.openweightlifting.org/search?name=${nameQuery}`);
```

Comment:
Reference the process.env.API variable like in the other fetch / call functions.

Reply:
ok changed thought it was a different domain intentionally so left it